casadm/cas_lib.c

```diff
@@ -786,6 +789,7 @@ struct cache_device *get_cache_device(const struct kcas_cache_info *info, bool b
 	cache->expected_core_count = info->info.core_count;
 	cache->id = cache_id;
 	cache->state = info->info.state;
+	cache->standby_detached = info->info.standby_detached;
 
 	if (strncpy_s(cache->device, sizeof(cache->device),
 			info->cache_path_name,
```

Comment:
What is the content of `info->cache_path_name` if cache is detached? I wonder if it wouldn't be safer to put `-` into `cache->device` instead of the unspecified(?) value.

Reply:
info->cache_path_name is string consisting of '\0's, len = 4096